lib/crypto/c_src/pbkdf2_hmac.c

```diff
@@ -22,10 +22,9 @@
 #include "pbkdf2_hmac.h"
 #include "digest.h"
 
-ERL_NIF_TERM pbkdf2_hmac_nif(ErlNifEnv* env, int argc,
-                             const ERL_NIF_TERM argv[])
+static ERL_NIF_TERM pbkdf2_hmac(ErlNifEnv* env, int argc,
+                                const ERL_NIF_TERM argv[])
```

Comment:
Should the entire function `pbkdf2_hmac()` be enclosed in `#ifdef HAS_PKCS5_PBKDF2_HMAC` maybe?

Reply:
Good idea, let's enclose it.

Reply:
Updated and enclosed it `#ifdef HAS_PKCS5_PBKDF2_HMAC ...`